src/components/Slides.vue

```diff
@@ -53,7 +56,7 @@ export default {
   data () {
     return {
       searchString: '',
-      events: [
+      slides: [
         {
           _id: '5a700ee29c61ea4f72cd23e2',
```

Comment:
I know you didn't add it, but should we have these premade slides in the repo?

Reply:
Well you can't do much with the search or the saved slides list if you don't have any data to display, I'm assuming  a later task will include replacing the hardcoded data with Sherbrooke's actual data

Reply:
It may be useful to have in there for now till we hook in the actual backend (The slides that will be coming in from firebase).